bridge/contracts/AToken.sol

```diff
@@ -34,13 +38,21 @@ contract AToken is
     function migrate(uint256 amount) public {
         require(_migrationAllowed, "MadTokens migration not allowed");
         IERC20(_legacyToken).transferFrom(msg.sender, address(this), amount);
-        _mint(msg.sender, amount);
+        _mint(msg.sender, _convert(amount));
     }
 
     function allowMigration() public onlyFactory {
         _migrationAllowed = true;
     }
 
+    function finishEarlyStage() public onlyFactory {
```

Comment:
Not sure about the naming here - I can see there is both a way to turn the mulitplier on/off via `toggleMultiplierOn()`/`finishEarlyStage()` and i assume in the "early stages" the tokens are multiplied, however seeing as the early stagy ending just toggles the multiplier off would it not be simpler to have a setter for the multiplier? Unless there are other things dependant on this "early stage" so it needs to be kept broad?